node/src/command.rs

```diff
@@ -272,9 +287,13 @@ pub fn run() -> Result<()> {
 		Some(Subcommand::FrontierDb(cmd)) => {
 			let runner = cli.create_runner(cmd)?;
 			runner.sync_run(|config| {
-				let params = crate::service::new_partial(&config, &cli.eth)?;
-				let client = params.client;
-				let (_, _, _, frontier_backend) = params.other;
+				let PartialComponents { client, other, .. } =
+					crate::service::new_partial(&config, &cli.eth)?;
+				let (_, _, _, frontier_backend, _) = other;
+				let frontier_backend = match frontier_backend {
+					fc_db::Backend::KeyValue(kv) => Arc::new(kv),
+					_ => panic!("Only fc_db::Backend::KeyValue supported"),
+				};
 				cmd.run(client, frontier_backend)
 			})
 		},
```

Comment:
Some code updates are missing in the None option below (see [here](https://github.com/substrate-developer-hub/substrate-parachain-template/pull/153/files#diff-8ce03a5a2b79f3021ff2d6322d13448db8cfd5fb9267517ab4b96d0d299ed45fL258-L281)).